### PR TITLE
Db changes in files

### PR DESCRIPTION
**Pull request by Abdullaharifbhatti**
Whoever pulls the port number need to change back again to 3306 in vagrant file

### CODE REVIEW

**Comment by oona988** [comment]
Some of these database changes are also needed in database routers. So that API requests are possible to create. For example in src/routers/api.js and src/routers/objectRouters/updates.js.

For example row 35 in updates.js file:

database.query("INSERT INTO Paivitykset(Tekija, Segmentti, Lumilaatu, Teksti, Aika) VALUES(?, ?, ?, ?, NOW())",

**Comment by oona988** [comment]
Also is it possible to store multiple snowtypes to database?

**Comment by oona988** [comment]
Some errors appear when creating database, seems that there is some mysql syntax problem: 

default: ERROR 1064 (42000) at line 26: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT tunniste PRIMARY KEY (Aika, Segmentti)
    default: )' at line 14
    default: ERROR 1136 (21S01) at line 8: Column count doesn't match value count at row 1
    default: ERROR 1146 (42S02) at line 1: Table 'pallas.Koordinaatit' doesn't exist
    default: ERROR 1146 (42S02) at line 1: Table 'pallas.Alalumilaadut' doesn't exist

**Comment by oona988** [comment]
> Some errors appear when creating database, seems that there is some mysql syntax problem:
> 
> default: ERROR 1064 (42000) at line 26: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT tunniste PRIMARY KEY (Aika, Segmentti) default: )' at line 14 default: ERROR 1136 (21S01) at line 8: Column count doesn't match value count at row 1 default: ERROR 1146 (42S02) at line 1: Table 'pallas.Koordinaatit' doesn't exist default: ERROR 1146 (42S02) at line 1: Table 'pallas.Alalumilaadut' doesn't exist

File: src/sql/luonnit.sql [hunk 1]
```diff
@@ -28,9 +28,14 @@ CREATE TABLE Paivitykset (
     Segmentti BIGINT UNSIGNED,
     Aika DATETIME,
     Lumilaatu INT,
-    Teksti TEXT,
+    Kuvaus TEXT,
+    Lumen_kuva BLOB,
+    Lumilaatu_ID bigint(20) unsigned,
```
Comment (oona988) [inline src/sql/luonnit.sql:33]: There are Lumilaatu and Lumilaatu_ID in table Paivitykset, and both of these are referring to snowtype ID.

File: src/sql/Alalumilaadut.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+(7,'Ohut korppu',3,3),
+(8,'Rikkoutuva korppu',3,2),
+(9,'Kantava korppu',3,4),
+(10,'jää',5,3),
+(11,'haurastuva jää',5,4),
+(13,'kastuva',4,5),
+(14,'saturoitunut',4,2),
+(15,'kiviä',10,NULL),
+(16,'oksia',10,NULL),
+(17,'kantoja',10,NULL);
```
Comment (oona988) [inline src/sql/Alalumilaadut.sql:17]: Would be good to capitalize all of these sub snowtype names

**Comment by oona988** [comment]
There seems to be still some errors when creating the database:

default: ERROR 1215 (HY000) at line 26: Cannot add foreign key constraint
default: ERROR 1146 (42S02) at line 1: Table 'pallas.Koordinaatit' doesn't exist
default: ERROR 1146 (42S02) at line 1: Table 'pallas.Alalumilaadut' doesn't exist

**Review by collandk — APPROVED** [review]
Everything is working in Vagrant as expected now

**Review by collandk — CHANGES_REQUESTED** [review]
Whoops, cancelling the approval. Oonas comment was forgotten to fix:
There are Lumilaatu and Lumilaatu_ID in table Paivitykset, and both of these are referring to snowtype ID.

**Review by collandk — APPROVED** [review]

**Comment by collandk** [comment]
All fixes are now done and to us everything seems to be working fine!  It would be good for someone else to verify that all tables are created successfully with all the wanted initial data. The person who approves this change can also merge it.

**Review by emilcalonius — APPROVED** [review]
Everything seemed to work fine for me so I am approving these changes.

**Pull request merged**